packages/ripple-binary-codec/src/enums/README.md

```diff
@@ -31,30 +33,70 @@ Sort code numbers are reused for fields of different types, but different fields
 
 The unique [Field ID](https://xrpl.org/serialization.html#field-ids) is prefixed before the field in the final serialized blob. The size of the Field ID is one to three bytes depending on the type code and the field codes it combines.
 
-### isVLEncoded
+#### isVLEncoded
 
 If true, the field is Variable Length encoded and [length-prefixed](https://xrpl.org/serialization.html#length-prefixing). The variable-length encoded fields are `STI_VL`/`Blob`, `STI_ACCOUNT`/`AccountID`, and `STI_VECTOR256`/`Vector256`.
 
-### isSerialized
+#### isSerialized
 
 Fields are serialized if they are not [one of these](https://github.com/ripple/rippled/blob/eaff9a0e6aec0ad077f118501791c7684debcfd5/src/ripple/protocol/impl/SField.cpp#L71-L78) or if they are not an SField.
 
 - https://github.com/ripple/ripple-binary-codec/blob/14e76e68ead7e4bcd83c942dbdc9064d5a66869b/src/enums/definitions.json#L832
 - https://github.com/ripple/rippled/search?utf8=%E2%9C%93&q=taker_gets_funded&type=
 
-### isSigningField
+#### isSigningField
 
 True unless the field is [specified with `SField::notSigning`](https://github.com/ripple/rippled/blob/eaff9a0e6aec0ad077f118501791c7684debcfd5/src/ripple/protocol/impl/SField.cpp#L198).
 
-## Transaction Results
+### Transaction Results
 
 See:
 
 - https://github.com/ripple/rippled/blob/develop/src/ripple/protocol/TER.h
 - https://xrpl.org/transaction-results.html
 
-TODO: Write a script to read rippled's source file and generate the necessary mapping.
+To generate a new definitions file from rippled source code, use this tool: https://github.com/RichardAH/xrpl-codec-gen
 
-## Transaction Types
+### Transaction Types
 
 See https://github.com/ripple/rippled/blob/develop/src/ripple/protocol/TxFormats.h
+
+# Defining Your Own Definitions
+
+If you're building your own side chain or amendment for the XRPL, you may need to create new definitions.
+
+To do that there are three high level things you need to do:
+1. Copy and modify the `definitions.json` file to have your definitions ([link](https://github.com/XRPLF/xrpl.js/blob/main/packages/ripple-binary-codec/src/enums/definitions.json)) as a baseline, or you can generate your own from rippled source code with [this tool](https://github.com/RichardAH/xrpl-codec-gen)
```

Comment:
We do not want to recommend that people edit their definitions file by hand - it's a pain and it's really easy to make mistakes.